### PR TITLE
CUG updates

### PR DESCRIPTION
**Pull request by hjoliver**
Some minor additions to the User Guide, to record things I've had to explain to users recently by email.

### CODE REVIEW

**Comment by hjoliver** [comment]
Yes that bothers me too!

I used to call <the entities in question> "the suite" or "cylc", but those terms are obviously very ambiguous.  And "suite server" (as in the software counterpart of suite client commands) is too easily confused with the suite host server (hardware).  I guess "suite server program" is most accurate, but that seems quite unwieldy.  "suite daemon" is true most of the time.  What do you call the thing?

Now that I've written it down, maybe "suite server program" isn't too bad?

**Comment by matthewrmshin** [comment]
> What do you call the thing?

I have used the phrases such as *suite process*, *suite server process*, *main suite process*, *running suite*. (None of these are satisfying.)

**Comment by hjoliver** [comment]
I have used those terms too.  But it has more than one process.  Do you not like "suite server program"?

**Comment by matthewrmshin** [comment]
*Suite server program* is the best of the lots.

File: doc/src/cylc-user-guide/cug.tex [hunk 1]
```diff
@@ -7172,6 +7267,67 @@ \subsection{Suite Run Databases}
...
+it wipes out prior run history, and it could re-run a significant number of
+tasks that had already completed. 
+
+To restart the suite, the critical Cylc files that must be restored are:
+
+\lstset{language=transcript}
+\begin{lstlisting}
+# On the suite host:
+~/cylc-run/<SUITE-NAME>/
+    <suite.rc>   # live suite definition (located here in Rose suites)
```
[the remark this answers is not in the document]
Comment (hjoliver) [reply doc/src/cylc-user-guide/cug.tex]: OK, I'll remove those right now...

**Comment by oliver-sanders** [comment]
Conflicts :(

**Comment by matthewrmshin** [comment]
(Now in conflicts.)

**Comment by hjoliver** [comment]
Rebased and deconflicted.

**Comment by hjoliver** [comment]
(over 'n' out - have a good weekend)

**Comment by matthewrmshin** [comment]
(@hjoliver I can see that it is half past midnight where you are. Have a good weekend!)

**Comment by matthewrmshin** [comment]
(#2473 should fix the failing test - which is currently a bit unstable.)

**Comment by hjoliver** [comment]
Tests passed, including the flaky one, and two reviews done ... so I'll merge this now.

**Pull request merged**